functions/bubbleSort.js

```diff
@@ -0,0 +1,18 @@
+export default function bubbleSort(array){
+  if(Array.isArray(array) && array.every(Number.isInteger)){
+    let temp
+    for(let i = 0; i < array.length; i++){
+      for(let j = 0; j < array.length - i; j++){
+        if(array[j]>array[j+1]){
```

Comment:
Spaces around `>` to be consistent with the code formatting in the rest of your repo.